tools/nnpackage_tool/model2nnpkg/model2nnpkg.sh

```diff
@@ -64,11 +69,18 @@ extension=${modelfile##*.}
 
 echo "Generating nnpackage "$name" in "$outdir""
 mkdir -p "$outdir"/"$name"/metadata
+
+if [ ! -z "$config_src" ] && [ -s "$config_src" ]; then
```

Comment:
`[ ! -z "$config_src" ]` condition might be redundant.
```suggestion
if [ -f "$config_src" ]; then
```

Reply:
@mhs4670go 

I also thought ` -z ` might be redundant. However, I am not sure it is equivalent.
Also,  your suggestion `-f` allows zero-size config file while `-s` will not allow zero-size config file.

Reply:
@glistening Actually, I wanted to check whether it is a directory or not. But just `-s` option seems enough I think. And, I think `-z` option is unnecessary here.
```bash
if [ -s "$config_src" ]; then
```

Reply:
`$config_src` is optional. it can be null if not given. `-z` is to check if this option is provided and config file is given.

Reply:
@seanshpark Right. s option checks what z option does as well.

Reply:
OK, `-s` seems to work